resolwe_bio/docker_images/dnaseq/packages-r.txt

```diff
@@ -0,0 +1,2 @@
+bioc::IRanges@2.28.0
```

Comment:
It would probably suffice to install existing versions. In what I believe to be the latest dnaseq image (6.2.0) I see the following versions:

```
chemut_1.0.1
Rsamtools_2.8.0
GenomicRanges_1.44.0
IRanges_2.26.0
```

These should also match the Bioconductor release of 3.13 (and not 3.14, which is the latest).